rotkehlchen/tests/fixtures/blockchain.py

```diff
@@ -102,6 +102,7 @@ def _initialize_and_yield_evm_inquirer_fixture(
     )
     if mocked_proxies is not None:
         mock_proxies(parent_stack, mocked_proxies)
+    inquirer.contracts.initialize_common_abis()
```

Comment:
again with this one. I told you emove this.

Reply:
I've been all the weekend fighting with this and forgot